srcs/Config/AllConfig/AllConfig.cpp

```diff
@@ -0,0 +1,57 @@
+#include "AllConfig.hpp"
+
+AllConfig::AllConfig(){}
+
+AllConfig::AllConfig(const AllConfig &other)
+{
+	this->_host_config = other._host_config;
+	this->_location_config = other._location_config;
+}
+
+AllConfig::AllConfig(const ServerConfig &host_config, const std::map<std::string, LocationConfig> &_location_config)
```

Comment:
[ Question ]

```suggestion
AllConfig::AllConfig(const ServerConfig &host_config, const std::map<std::string, LocationConfig> &location_config)
```
引数の`_`は不要？

Reply:
修正済み！